compat/krb5.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 1996-2022 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef _SQUID__COMPAT_KRB5_H_
+#define _SQUID__COMPAT_KRB5_H_
```

Comment:
We are not consistent across the code base, but the PR spelling is less frequently used and, IMO, is inferior to its simpler/safer equivalents that do not start and end with underscores (that resemble various "reserved" names we should not be using):

```suggestion
#ifndef SQUID_COMPAT_KRB5_H
#define SQUID_COMPAT_KRB5_H
```

There is one more macro name at the end of the file.